scripts/migration/migrate_personal_to_profile_websites.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Script to migrate single value personal to profile_websites list."""
+
+import logging
+
+from modularodm import Q
+from website import models
+from website.app import init_app
+
+
+logger = logging.getLogger(__name__)
+
+
+def main():
+    init_app(routes=False)
+    logger.info("migrating personal to profileWebsites")
+    all_users = 0
+    migrated_users = 0
+    for user in get_users_with_social_field():
+        all_users += 1
+        if not user.social.get('profileWebsites', None):
+            personal = user.social.get('personal', None)
+            user.social['profileWebsites'] = [personal] if personal else []
+            migrated_users += 1
+        logger.info("{}'s social dictionary is now {}".format(user.social, user.fullname))
+        user.save()
```

Comment:
This might raise a ValidationError, e.g. if a user has an invalid personal website to begin with. The script should continue running, with the error logged:

```python
        try:
            user.save()
        except modularodm.exceptions.ValidationError as err:
            logger.error('Validation error occurred while trying to save user {}'.format(user._id))
            logger.error(str(err))
            logger.error('Continuing to next user...')
```